database.sql

```diff
@@ -588,12 +588,12 @@ CREATE TABLE IF NOT EXISTS `hook` (
 --
 -- TABLE host
 --
-CREATE TABLE IF NOT EXISTS `host` (
-	`id` tinyint unsigned NOT NULL auto_increment COMMENT 'sequential ID',
+CREATE TABLE IF NOT EXISTS `hosts` (
+	`id` mediumint unsigned NOT NULL auto_increment COMMENT 'sequential ID',
```

Comment:
`smallint` really is enough. This does cover 65,535 hostnames. `Mediumint` does cover 16,777,215 - which is much too much. This field will be used in tables like the workerqueue that is used in a high frequency, so I want to save as many space as possible.

Reply:
`as many space` => `as much space`

`as much` needs to be used with innumerables (like space, weight, etc...) while `as many` needs to be used with numerables (like people, apples, etc...)